instrumentation/othttp/handler.go

```diff
@@ -36,12 +38,15 @@ type Handler struct {
 	handler   http.Handler
 
 	tracer            trace.Tracer
+	meter             metric.MeterMust
 	propagators       propagation.Propagators
 	spanStartOptions  []trace.StartOption
 	readEvent         bool
 	writeEvent        bool
 	filters           []Filter
 	spanNameFormatter func(string, *http.Request) string
+	counters          map[string]*metric.Int64Counter
```

Comment:
There is no need for these to be pointers. You could just refer to the structs. This makes me wonder why not just list the four instruments as four fields. Why use a map in the first place?

Reply:
I think it reasonable to expect that more instruments will be added and putting them in a map avoids the need to update the structure with each new instrument.

Reply:
fixed pointers